scipy/fft/_pocketfft/version.md

```diff
@@ -0,0 +1,11 @@
+
+pypocketfft version
+-------------------
+
+SciPy currently vendors [pypocketfft][repo] at commit hash:
```

Comment:
it would be good to add the date of the commit here, too, so no investigation is needed to tell how far out of date we are time-wise

Reply:
I've added the full git log entry